src/Core/MailTemplates/Handlebars/LicenseExpired.text.hbs

```diff
@@ -0,0 +1,9 @@
+﻿{{#>BasicTextLayout}}
+{{#if IsOrganization}}
+This email is to notify you that your Bitwarden organization license has expired and must be updated for continued use. See the following article for details about replacing your license file:
+{{else}}
+This email is to notify you that your Bitwarden premium license has expired and must be updated for continued use. See the following article for details about replacing your license file:
+{{/if}}
+
```

Comment:
Newline spacing seems a little odd here. Is this correct? Not sure how the conditional keyword lines get rendered.

Reply:
It was done by hand.  It looked weird to me when I matched the newlines in the other examples (since the presence of an `else` condition means we don't have to put a newline above each instance of `This email is to...` and instead can rely on a single newline before the `if` statement).  I can rework to match the others exactly if you prefer.